vendor/github.com/SpectoLabs/raymond/helper.go

```diff
@@ -389,6 +390,57 @@ func eachHelper(context interface{}, options *Options) interface{} {
 	return result
 }
 
+func firstHelper(context interface{}, options *Options) interface{} {
```

Comment:
change to `vendor` package will get wiped the next time we run `go mod vendor`, so we need this change committed to `https://github.com/SpectoLabs/raymond` and then update `go.mod` to include the latest version of our raymond folk.

Reply:
Thanks, I will do so

Reply:
This I haven't done yet - I will asap though

Reply:
your change in raymond is merged, you can include it in the go mod here and do go mod vendor

Reply:
Have done so Tommy - thanks for guidance

Reply:
you need to literally run `go mod vendor` on the project to bring in the change, but never mind I will do it on master.